util/cron/common-arkouda-hpe-apollo-hdr.bash

```diff
@@ -5,3 +5,4 @@
 PYTEST_TEMP_DIR=/hpelustre/chapelu/arkouda-scratch/$CHPL_NIGHTLY_TEST_CONFIG_NAME
 rm -rf $PYTEST_TEMP_DIR; mkdir -p $PYTEST_TEMP_DIR
 export CHPL_TEST_ARKOUDA_PYTEST_OPTIONS="--temp-directory=$PYTEST_TEMP_DIR"
+export ARKOUDA_DEFAULT_TEMP_DIRECTORY=$PYTEST_TEMP_DIR
```

Comment:
With this change, I have a mild preference for using `CHPL_TEST_ARKOUDA_TEMP_DIR` for the name instead of `PYTEST_TEMP_DIR`, mostly because the current benchmark infrastructure doesn't use pytest. But, definitely not a showstopper.